### DISCUSSION TITLE
Wrong explorer link to deployment

### DISCUSSION THREAD

**Discussion by WayneWayner** [subquery/subql#934] [Frontier TestNet - Bug Reporting]
If I use the link to an old deployment it brings me to a newer one. 
Maybe you should add the deployment ID to the link to avoid any confusions. 

![grafik](https://user-images.githubusercontent.com/60453944/165631081-6bf6dcdc-2b5e-4016-adcc-2a3edd484e96.png)

Clicking the old kusama deployment (QmVcksANdHhnA2YEguJ8LvsftNkipTwkgV9eJyDPZFUqHm) at the bottom gets me to
https://testnet.thechaindata.com/explorer/project/0x01/overview

![grafik](https://user-images.githubusercontent.com/60453944/165631174-4749a255-ae34-426d-91dd-54392b5a3863.png)
This should be the correct link:
https://testnet.thechaindata.com/explorer/project/0x01/overview?deploymentId=QmVcksANdHhnA2YEguJ8LvsftNkipTwkgV9eJyDPZFUqHm

**Comment by mzxyz** [comment]
@CaiYiLiang

**Comment by lilian79511** [comment]
This is reported as fixed by @CaiYiLiang. Discussion will be locked.
